.github/workflows/publish-noir-js.yml

```diff
@@ -17,9 +17,13 @@ jobs:
       - name: Install jq
         run: sudo apt-get install jq
 
+      - name: Setup toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: 1.67.0
```

Comment:
Why the bump to 1.67.0?

Reply:
rustc would throw an error when installed on 1.66.1:

https://github.com/noir-lang/noir/actions/runs/6375729739/job/17301824921